packages/flutter/test/cupertino/text_selection_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter/cupertino.dart';
+
+void main() {
```

Comment:
Thanks for creating these two test files 👍

Reply:
Thank you for review!